flink-formats/flink-csv/src/main/java/org/apache/flink/formats/csv/CsvRowSerializationSchema.java

```diff
@@ -319,7 +323,12 @@ private static RuntimeConverter createRuntimeConverter(TypeInformation<?> info)
             return (csvMapper, container, obj) -> container.textNode(obj.toString());
         } else if (info.equals(Types.LOCAL_DATE_TIME)) {
             return (csvMapper, container, obj) ->
-                    container.textNode(DATE_TIME_FORMATTER.format((LocalDateTime) obj));
```

Comment:
delete useless DATE_TIME_FORMATTER

Reply:
I noticed that this `SQL_TIMESTAMP_FORMAT` is not exactly the same with `DATE_TIME_FORMATTER`, please double check this won't cause any issue.

Reply:
I checked the change is safe, although `DATE_TIME_FORMATTER `  supports `2020-01-01 12:18` and `2020-01-01 12:18:00`, but the deserializer uses `Timestamp.valueOf(tiemstampString)`  which only supports `2020-01-01 12:18:00`, currently, and all serializer use `LocalDateTime` object which won't produce timestamp string like `2020-01-01 12:18`.